packages/@atjson/document/src/serialize.ts

```diff
@@ -752,15 +755,6 @@ export function deserialize<A>(
 
   for (let i = 0, len = positions.length - 1; i < len; i++) {
     let block = blocks[i];
-    if (block.type === "text") {
-      annotations.push(
-        new ParseAnnotation({
-          start: positions[i],
-          end: positions[i] + 1,
-        })
-      );
-      continue;
-    }
```

Comment:
ah what was going on here?

Reply:
I can separate this into another PR if that helps